lib/rust/parser/doc-parser/src/doc_sections.rs

```diff
@@ -155,6 +186,9 @@ impl<L> TokenConsumer<L> for DocSectionCollector {
     fn enter_keyed_section(&mut self, header: Span<'_, L>) {
         self.finish_section();
         let key = header.to_string();
+        if key.to_lowercase() == "arguments" {
```

Comment:
```suggestion
        if key.eq_ignore_ascii_case("Arguments") {
```
We process a lot of documentation; it's important to avoid allocations when we can.